src/ci/bin/test.inc.sh

```diff
@@ -810,7 +811,11 @@ cromwell::private::pip_install() {
     pip_package="${1:?pip_install called without a package}"; shift
 
     if [[ "${CROMWELL_BUILD_IS_CI}" == "true" ]]; then
-        sudo -H "${PYTHON3_HOME}/bin/pip" install "${pip_package}" "$@"
+        if [[ "${CROMWELL_BUILD_PROVIDER}" == "${CROMWELL_BUILD_PROVIDER_CIRCLE}" ]]; then
+            pip3 install "${pip_package}" "$@"
+        else
+            sudo -H "${PYTHON3_HOME}/bin/pip" install "${pip_package}" "$@"
+        fi
```

Comment:
ToL: I'm surprised `sudo` doesn't work on circle, or is this some other inconsistency like needing to `update-alternatives` for `pip`? Assuming circle does have sudo, something generic like `sudo -H "$(command -v pip)" install "${pip_package}" "$@"` should work on all CI systems 🤞

Reply:
sudo worked, but default pip on Ubuntu 20.04 was referring to python2 version. Once we drop Travis `pip3 install "${pip_package}" "$@"` will be the only version.
Also turned out, sudo was not required here.

Reply:
> will be the only version

...until we move with the rest of DSP to GitHub actions 😉